src/components/BlogPostTags.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div class="tags-container">
+    <button @click="toggleInputBar" class="new-tag-button">
+      <FontAwesomeIcon :icon="['fas', 'plus']" /> New Tag
+    </button>
+    <div v-for="tag in props.tags" :key="tag" class="tag-button">
+      {{ tag }}
+      <font-awesome-icon
```

Comment:
```suggestion
      <FontAwesomeIcon
```
We use PascalCase rather kebab-case